packages/pilot/src/containers/Sidebar/index.js

```diff
@@ -158,10 +160,9 @@ class SidebarContainer extends React.Component {
             justifyContent="center"
           >
             <Button
-              onClick={() => {
-                // eslint-disable-next-line no-undef
-                window.open('https://dashboard.pagar.me/')
-              }}
+              onClick={() => window.open(
```

Comment:
aqui ta uns problemas de lint que esse codigo abaixo resolve. Porem talvez seja interessante criar uma função pra esse onClick e chamar ela ali para não ficar essa linha grande demais
```suggestion
              // eslint-disable-next-line no-undef
              onClick={() => window.open(`https://dashboard.pagar.me/#login?session_id=${sessionId}&redirect_to=dashboard.home&environment=${environment}`)}
```